lib/babe/verify_test.go

```diff
@@ -671,6 +673,144 @@ func Test_verifier_verifyAuthorshipRight(t *testing.T) {
 	}
 }
 
+func Test_verifier_verifyBlockEquivocation(t *testing.T) {
+	ctrl := gomock.NewController(t)
```

Comment:
It would be nice to have a controller + mocks configured per subtest as described in https://engineering-handbook.onrender.com/development/tech-stack/go#mocks-with-subtests instead of in the parent test for the subtests